### PR TITLE
wrote integration test for api route and passed the test

### PR DESCRIPTION
**Pull request by JeehyaeDance**
added prettier for code formatting. There are little changes made on every file with prettier. Also installed jest and supertest to write integration test to check api route is working. please ignore reviews.js since I was writing the file to learn jest.

### CODE REVIEW

File: database_reviews/fake-data.js [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 //generating 100 set of reviews
 for (let i = 0; i < 100; i++) {
-    let numberOfReviews = faker.random.number({min: 1, max: 20});
-    let numberOfWords = faker.random.number({min: 5, max: 15});
-    for (let j = 0; j < numberOfReviews; j++) {
-        reviews.push(`"${faker.random.words(numberOfWords)}"`, faker.random.number({min: 1, max: 5}), 
-        faker.random.number({min: 0, max: 1}), faker.name.firstName(), faker.date.recent(), i)
-    }
+  let numberOfReviews = faker.random.number({ min: 1, max: 20 });
+  let numberOfWords = faker.random.number({ min: 5, max: 15 });
```
Comment (thomasij813) [inline database_reviews/fake-data.js]: Is there a reason why you're determining the number of words each review should have outside the second for loop? By doing so, aren't you guaranteeing that for each review for a given product will have exactly number of words?

File: package.json [hunk 2]
```diff
@@ -8,10 +8,12 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "webpack -d --watch & nodemon ./server/index.js",
-    "seed": "node ./database_reviews/fake-data.js"
+    "start": "webpack -d --watch & nodemon ./server/server.js",
+    "seed": "node ./database_reviews/fake-data.js",
```
Comment (thomasij813) [inline package.json]: This script creates the import file, but how do you actually get it into mysql? I would update this script so it does both the creating of the file followed by the import, or include instructions in your readme about what to do once the seed data has been generated (ie, how to import).

File: package.json [hunk 3]
```diff
@@ -8,10 +8,12 @@
...
+    "test": "jest"
   },
   "dependencies": {
```
Comment (thomasij813) [inline package.json]: Have you thought about separating using `devDependencies` for things like webpack and babel? Basically, in your `package.json` file, you want to separate your dependencies that are needed for actually running the application (react, express, body-parser, etc.) from those that are needed for building/compiling your code (babel, webpack) and/or setting up your dev environment (jest, supertest). The dependencies that fall into that second category, go in your `devDependencies`.

Here's some resources that talk about why you might do this and how to achieve it as you bring in new dependencies to your project:

https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file
https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c

File: reviews.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const db = require("./database_reviews/database.js");
+
+router.get("/reviews", async (req, res) => {
+  const data = await db.query("SELECT * FROM reviews");
+  return res.json(data);
+});
+
+module.exports = router;
```
Comment (thomasij813) [inline reviews.js]: Are you using this code at all? I might be wrong, but from what I can tell you're not actually importing this file anywhere, so I'm not sure if it's serving any purpose.

File: spec/api.test.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  await data.query("DROP TABLE reviews");
+  data.end();
+});
+
+// describe("sample test", () => {
+//   test("anything", function() {
+//     console.log("env", process.env.NODE_ENV);
+//     expect(true).toEqual(true);
+//   });
+// });
```
Comment (thomasij813) [inline spec/api.test.js]: Remove these comments :p

File: spec/api.test.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  await data.query(
+    "CREATE TABLE reviews (id INT NOT NULL AUTO_INCREMENT, text VARCHAR(255) NOT NULL, star INT, verified TINYINT(1), name VARCHAR(255), date VARCHAR(255), product_id INT NOT NULL, PRIMARY KEY (id))"
+  );
+});
+
+beforeEach(async () => {
+  await data.query(
+    "LOAD DATA LOCAL INFILE 'testReview.csv' INTO TABLE reviews FIELDS TERMINATED BY ',' LINES TERMINATED BY ',' (text, star, verified, name, date, product_id);"
+  );
+});
```
Comment (thomasij813) [inline spec/api.test.js]: Space here to visually separate the `beforeEach` from the `afterEach`

Comment (thomasij813) [inline spec/api.test.js]: Is there a reason why you're using `beforeEach`, `beforeAll`, `afterEach` and `afterAll` when you only have one test? Can't you just use either `beforeEach` and `afterEach` OR `beforeAll` and `afterAll`?

File: database_reviews/fake-data.js (hunk 1 above)
Comment (JeehyaeDance) [reply database_reviews/fake-data.js]: Oh I did not notice this at all. Yes, you are absolutely right. I need to fix that. thank you!

File: spec/api.test.js (hunk 6 above)
Comment (JeehyaeDance) [reply spec/api.test.js]: I plan to write more tests later, so it would be helpful to keep them for now

**Pull request merged**